src/components/PageSwitcher/components/EEGEduHeartSpectra/EEGEduHeartSpectra.js

```diff
@@ -94,48 +93,81 @@ export function setup(setData, Settings) {
 export function renderModule(channels) {
   function renderCharts() {
     return Object.values(channels.data).map((channel, index) => {
-        const options = {
-          ...generalOptions,
-          scales: {
-            xAxes: [
-              {
-                scaleLabel: {
-                  ...generalOptions.scales.xAxes[0].scaleLabel,
-                  labelString: specificTranslations.xlabel
-                }
+      const options = {
+        ...generalOptions,
+        scales: {
+          xAxes: [
+            {
+              scaleLabel: {
+                ...generalOptions.scales.xAxes[0].scaleLabel,
+                labelString: specificTranslations.xlabel
               }
-            ],
-            yAxes: [
-              {
-                scaleLabel: {
-                  ...generalOptions.scales.yAxes[0].scaleLabel,
-                  labelString: specificTranslations.ylabel
-                },
-                ticks: {
-                  min: 0
-                }
+            }
+          ],
+          yAxes: [
+            {
+              scaleLabel: {
+                ...generalOptions.scales.yAxes[0].scaleLabel,
+                labelString: specificTranslations.ylabel
+              },
+              ticks: {
+                min: 0
               }
-            ]
-          },
-          elements: {
-            point: {
-              radius: 3
             }
-          },
-          title: {
-            ...generalOptions.title,
-            text: generalTranslations.channel + 
-              channelNames[1] + 
-              " - Estimated HR: " +
-              channel.peakF + " BPM"
+          ]
+        },
+        elements: {
+          point: {
+            radius: 8
           }
-        };
-
-        return (
-          <Card.Section key={"Card_" + index}>
-            <Line key={"Line_" + index} data={channel} options={options} />
-          </Card.Section>
-        );
+        },
+        animation: {
+          duration: 500
+        },
+        title: {
+          ...generalOptions.title,
+          text: generalTranslations.channel + 
+            channelNames[1] + 
+            " - Estimated HR: " +
+            channel.peakF + " BPM"
+        }
+      };
+
+      if (index === 0) {
+        if (channel.xLabels) {
```

Comment:
computing peak here on this single channel, then making new data set with spectra as one and single point as the other